custom_components/unifiprotect/services.yaml

```
@@ -1,3 +1,32 @@
+generate_data:
```

```suggestion
take_sample:
```

Maybe this is a bit more descriptive ?